src/routines/level3/xgemm.cpp

```diff
@@ -33,10 +33,11 @@ Xgemm<T>::Xgemm(Queue &queue, EventPointer event, const std::string &name):
     #include "../../kernels/level3/convert_symmetric.opencl"
     #include "../../kernels/level3/convert_triangular.opencl"
     #include "../../kernels/level3/convert_hermitian.opencl"
+    }, {
+    // keep in mind C1091 in MSVC 2013
```

Comment:
Isn't this tested in AppVeyor? Or did they lift that limitation in newer versions of MSVC++?